drivers/net/bnxt/bnxt_hwrm.c

```diff
@@ -2263,6 +2266,29 @@ void vf_vnic_set_rxmask_cb(struct bnxt_vnic_info *vnic, void *flagp)
 	vnic->flags = *flag;
 }
 
+static void bnxt_vnic_count(struct bnxt_vnic_info *vnic, void *cbdata)
+{
+	uint32_t *count = cbdata;
+
+	if (vnic->func_default)
+		*count = *count + 1;
```

Comment:
The VNIC count is currently *always* non-zero, even when there are no VNICs allocated.  See CTRL-25328 for details.